### PR TITLE
MSFT Contributions June 2020

### PR DESCRIPTION
**Pull request by jankrassnigg**

### CODE REVIEW

File: Code/Engine/Texture/TexConv/Implementation/TextureModifications.cpp [hunk 1]
```diff
@@ -26,7 +26,7 @@ ezResult ezTexConvProcessor::ForceSRGBFormats()
   return EZ_SUCCESS;
 }
 
-ezResult ezTexConvProcessor::GenerateMipmaps(ezImage& img, ezUInt32 uiNumMips, MipmapChannelMode channelMode) const
```
Comment (C-Core) [inline Code/Engine/Texture/TexConv/Implementation/TextureModifications.cpp:29]: This is not correct and overwriting a change made on the ez side

File: Code/Engine/Texture/TexConv/TexConvProcessor.h [hunk 2]
```diff
@@ -48,14 +48,7 @@ class EZ_TEXTURE_DLL ezTexConvProcessor
   ezResult PremultiplyAlpha(ezImage& image) const;
   ezResult DilateColor2D(ezImage& img) const;
   ezResult Assemble2DSlice(const ezTexConvSliceChannelMapping& mapping, ezUInt32 uiResolutionX, ezUInt32 uiResolutionY, ezColor* pPixelOut) const;
-
-  enum class MipmapChannelMode
```
Comment (C-Core) [inline Code/Engine/Texture/TexConv/TexConvProcessor.h:52]: here as well, this change was introduced on the ez side

**Review by C-Core — CHANGES_REQUESTED** [review]

File: Code/Engine/Texture/TexConv/Implementation/TextureModifications.cpp (hunk 1 above)
Comment (jakrams) [reply Code/Engine/Texture/TexConv/Implementation/TextureModifications.cpp:29]: fixed

File: Code/Engine/Texture/TexConv/TexConvProcessor.h (hunk 2 above)
Comment (jakrams) [reply Code/Engine/Texture/TexConv/TexConvProcessor.h:52]: fixed

**Review by C-Core — APPROVED** [review]

**Pull request merged**